Common/utils.ts

```diff
@@ -281,4 +274,20 @@ export function getTotalCS(playerStatsModel: AggregatedPlayerStatModel | PlayerG
 
 export function deepCopy<T>(object: T): T {
   return JSON.parse(JSON.stringify(object)) as T;
+}
```

Comment:
Just wondering why is this in the front-end and backend common?

Reply:
idk it was there before, i just didnt touch it

Reply:
????